front/package.json

```diff
@@ -18,15 +18,16 @@
     "@types/react-dom": "^17.0.11",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-rounded-image": "^2.0.14",
```

Comment:
jeśli jest nieużywany, to wyrzucić go. Niech niepotrzebnie się nie dociągają dodatkowe kilobajty.